app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<header>
+  <h1>Welcome to the Tasklist</h1>
+  <p> Here are the things to do<p>
+  </header>
+
+
+  <main class = "main-container">
```

Comment:
While the style we use for Ruby includes the spaces in `var = 1`, in HTML, we leave those extra spaces out `class="main-container"`